app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,26 @@
+class PasswordResetsController < ApplicationController
+  def show
+    user = User.where(token: params[:id]).first
```

Comment:
You should use `find_by` instead of `where(...).first`.

See above mentioned Homework.